src/Arcus.Messaging.Pumps.ServiceBus/Configuration/IAzureServiceBusQueueMessagePumpOptions.cs

```diff
@@ -47,11 +48,13 @@ public interface IAzureServiceBusQueueMessagePumpOptions
         /// <summary>
         /// Gets the options to control the correlation information upon the receiving of Azure Service Bus messages in the <see cref="AzureServiceBusMessagePump"/>.
         /// </summary>
+        [Obsolete("Will be moved to message routing options in the future")]
```

Comment:
The problem here is that we have interfaces to expose subsections of the options to users (like: differentiate between queues and topics), but these interfaces should be updated with the message routing instead of (currently) the message routing properties.

So, now, we have `MessageRouterOptions.Correlation` but we expose it in the interfaces as `Correlation`. If we in the next major add a `Routing` property to the interfaces, it will be more maintainable for us.